crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -957,7 +978,23 @@ impl<Codec: RawEntryCodec> StateMachine<Codec> {
                 )
                 .await?
             }
-            _ => {
+            InvocationStatus::Scheduled(scheduled) => {
+                self.terminate_scheduled_invocation(
+                    ctx,
+                    TerminationFlavor::Cancel,
+                    invocation_id,
+                    scheduled,
+                )
+                .await?
+            }
+            InvocationStatus::Killed(_) => {
+                // Nothing to do here really, let's send again the abort signal to the invoker just in case
+                Self::do_send_abort_invocation_to_invoker(ctx, invocation_id, true);
```

Comment:
Maybe log that we received a cancel command for a killed invocation and that the cancel command won't have an effect.